runtime/src/lib.rs

```diff
@@ -255,6 +258,17 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
 }
 
+type MaxRecordContentLength = ConstU32<300>;
+type SignatureLength = ConstU32<150>;
```

Comment:
I guess we will change this depending on the encryption method we use later on?

Reply:
Yep, these numbers are placeholders at the moment.

Reply:
Yep, these numbers are placeholders at the moment.